src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -1259,9 +1259,14 @@ public async Task<(long ClusterEtag, string ClusterId, long newIdentityValue)> G
             var (_, identityInfoResult) = await SendToLeaderAsync(new IncrementClusterIdentitiesBatchCommand(databaseName, ids));
 
             var identityInfo = identityInfoResult as List<long> ?? throw new InvalidOperationException(
-                                   $"Expected to get result from raft command that should generate a cluster-wide batch identity, but didn't. Leader is {LeaderTag}, Current node tag is {NodeTag}.");
+                    $"Expected to get result from raft command that should generate a cluster-wide batch identity, but didn't. Leader is {LeaderTag}, Current node tag is {NodeTag}.");
 
-            return identityInfo;
+            var rc = new List<long>();
+            foreach (var identity in identityInfo)
+            {
+                rc.Add((long)identity);
```

Comment:
Why is the cast required? Or even all of this code? You are reading from `List<long>` _to_ `List<long>`